src/Message/Security/SendScanResult.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Message\Security;
+
+final class SendScanResult
+{
+    private string $organizationAlias;
+    private string $packageName;
+    private string $packageId;
+
+    /**
+     * @var mixed[]
+     */
+    private array $result;
+
+    /**
+     * @param mixed[] $result
+     */
+    public function __construct(string $organizationAlias, string $packageName, string $packageId, array $result)
```

Comment:
Maybe we can add `emails` array to this message. This will simplify the handler. You can easliy add `ownersEmails()` method to `Package` entity